11LessonEleven/homework/css/inspired.css

```diff
@@ -0,0 +1,98 @@
+main{
+  font-size: serif;
+}
+
+.headline{
+  background-image: url("../img/bg2.jpg");
+  background-attachment: fixed;
+  background-size: cover;
+  color:#fff;
+  display: flex;
+  flex-flow:column nowrap;
+  align-items: flex-start;
+  justify-content: center;
+  height: 500px;
+}
+
+.headline h1{
+  margin-left: 100px;
+  width:400px;
```

Comment:
Width (or font-size) needs to be responsive; it runs of the edge of the screen @ 320px.